examples/grpc-server-with-prometheus/client/client.go

```diff
@@ -22,13 +46,37 @@ func main() {
 
 	defer conn.Close()
 
+	// Create a HTTP server for prometheus.
+	httpServer := &http.Server{Handler: promhttp.HandlerFor(reg, promhttp.HandlerOpts{}), Addr: fmt.Sprintf("0.0.0.0:%d", 9094)}
+
+	// Start your http server for prometheus.
+	go func() {
+		if err := httpServer.ListenAndServe(); err != nil {
+			log.Fatal("Unable to start a http server.")
+		}
+	}()
+
 	// Create a gRPC server client.
 	client := pb.NewDemoServiceClient(conn)
-	// Call “SayHello” method and wait for response from gRPC Server.
-	resp, err := client.SayHello(context.Background(), &pb.HelloRequest{Name: "Test"})
-	if err != nil {
-		log.Fatal(err)
+	fmt.Println("Start to call the method called SayHello every 3 seconds")
+	go func() {
+		for {
+			// Call “SayHello” method and wait for response from gRPC Server.
+			_, err := client.SayHello(context.Background(), &pb.HelloRequest{Name: "Test"})
+			if err != nil {
+				log.Fatal(err)
+			}
+			// Increase the count of calling of SayHello
+			customizedCounterMetric.WithLabelValues("Test").Inc()
```

Comment:
wouldn't it make more sense to actually use the go-grpc-prometheus client metrics?

Reply:
Ok. I will revise it soon